index.htm

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Biltong Kindom</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Kindly write the script link either in the head section or at the bottom of your body tag as this way it will affect the speed at which the page loads.
please visit the link below to learn more about this.
https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup